samples/subsys/mgmt/hawkbit/sample.yaml

```diff
@@ -17,6 +17,8 @@ tests:
   sample.net.hawkbit.default: {}
   sample.net.hawkbit.sysbuild:
     sysbuild: true
+    extra_args:
+      - platform:stm32h573i_dk/stm32h573xx:mcuboot_CONFIG_BOOT_MAX_IMG_SECTORS=128
```

Comment:
I would prefer to see it in a sysbuild and board specific overlay (https://docs.zephyrproject.org/latest/build/sysbuild/index.html#zephyr-application-configuration-directory) instead of here, so building this sample outside of the test won't fail too

Reply:
I originally used this approach, but once a full application configuration directly is supplied sysbuild will stop using the original application directory for configuration, so we would have to duplicate a lot of configuration for each board (IE board specific overlays)